node/src/chain/acceptor.rs

```diff
@@ -167,7 +167,12 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
         if mrb_height > 0 && mrb_state_hash != state_root {
             info!("revert to last finalized state");
             // Revert to last known finalized state.
-            acc.try_revert(RevertTarget::LastFinalizedState).await?;
+            if let Err(err) =
+                acc.try_revert(RevertTarget::LastFinalizedState).await
+            {
+                warn!("Reverting to last epoch: Failed to revert to last finalized state: {err}");
```

Comment:
There are some case when a revert fails for valid reason (like target iteration greater then the current one)
I would leave the revert strategy out from this PR

Reply:
Nevermind I was wrong
Those kind of checks are not part of `Acceptor::try_revert` but instead are part of `Fallback::try_revert`

Probably it worths to rename `Acceptor::try_revert` into `Acceptor::revert`